packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -140,20 +142,24 @@ export const Query = {
       ...categories,
     ]
 
+    const validCollections = collections
+      // Nullable slugs may cause one route to override the other
+      .filter((node) => Boolean(StoreCollection.slug(node, null, ctx, null)))
+
     return {
       pageInfo: {
-        hasNextPage: false,
-        hasPreviousPage: false,
+        hasNextPage: validCollections.length - after > first,
+        hasPreviousPage: after > 0,
         startCursor: '0',
-        endCursor: '0',
+        endCursor: (
```

Comment:
there's only one page on collections right? then these pagination things are always `false`, aren't they?

Reply:
Not anymore :)